lib/page/all_countries.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../world.rb'
+
+module Page
+  class AllCountries
+    attr_accessor :world
+    def initialize
+      @world = World.new.as_json.to_a
```

Comment:
Why are you converting this to an Array? The underlying structure is Hash-like, so seems like it would simpler to use that rather than flattening to a very odd interleaved array.